gb-tracy/README.md

```diff
@@ -0,0 +1,25 @@
+# Lab 03: jQuery Events
+
+**Author**: Tracy Williams Tim Busch
+**Version**: 1.3.0 
+
+## Overview
+<!-- Provide a high level overview of what this application is and why you are building it, beyond the fact that it's an assignment for a Code Fellows 301 class. (i.e. What's your problem domain?) -->
+This application builds a tab navigable blog that takes raw data in the form of articles with key value pairs and using jQuery to traverse and fill the DOM with that data and displays them on a page. 
+
+## Getting Started
+<!-- What are the steps that a user must take in order to build this app on their own machine and get it running? -->
+They would need to fork the application from GitHub, clone down the link from their own github, and open in index.html
```

Comment:
Instead of "They would", this should be in the form of directions to a user: "Start by forking this application from GitHub. Clone that URL from your  account and open the index.html file."